### DISCUSSION TITLE
Irregular separators

### DISCUSSION THREAD

**Discussion by StephanZilke** [Live-Charts/LiveCharts2#404] [Q&A]
Hi,
is there a possibility to define which separator lines are displayed?
My use case: I want to display a line series over a whole year and only show the separator lines for the first of the month.

**Comment by langejay** [comment]
Not sure how to set the FIRST separator line (probably uses the minimum value in the data so you may need to add a hidden value into your data if it doesnt contain a good starting point), but the GAP between the separators can be set by using:

`      XAxis = (Axis)myChart.XAxes.First();
       XAxis.MinStep = 100000;`

Hope this helps in some way.
